homeassistant/components/vulcan/calendar.py

```diff
@@ -63,12 +62,8 @@ def __init__(self, client, data, entity_id) -> None:
         self.client = client
         self.entity_id = entity_id
         self._unique_id = f"vulcan_calendar_{self.student_info['id']}"
-        if data["students_number"] == 1:
-            self._attr_name = "Vulcan calendar"
-            self.device_name = "Calendar"
-        else:
-            self._attr_name = f"Vulcan calendar - {self.student_info['full_name']}"
-            self.device_name = f"{self.student_info['full_name']}: Calendar"
+        self._attr_name = f"Vulcan calendar - {self.student_info['full_name']}"
+        self.device_name = f"{self.student_info['full_name']}: Calendar"
```

Comment:
`self.device_name` is only used in the device info below so we can change `self.device_name` to be a local variable or even better just set the name directly inside device info dict.